libs/labelbox/src/labelbox/schema/ontology.py

```diff
@@ -149,6 +155,7 @@ class Scope(Enum):
     schema_id: Optional[str] = None
     feature_schema_id: Optional[str] = None
     scope: Scope = None
+    ui_mode: UIMode = None # How this classification should be answered (e.g. hotkeys / autocomplete, etc)
```

Comment:
```suggestion
    ui_mode: Optional[UIMode] = None # How this classification should be answered (e.g. hotkeys / autocomplete, etc)
```
This is the pattern we follow (I would say for typing clarity)

Reply:
Done!